common/source/docs/common-rc-transmitter-flight-mode-configuration.rst

```diff
@@ -97,18 +97,21 @@ The transmitter must emit PWM signals in the correct range to allow us
 to map a mode to a switch position.
 
 .. note::
-
+   Generally the range is between 1000 to 2000 us (1.0 to 2.0 ms).
+   ms = millisecond
```

Comment:
these will be on the above line...RST ignores single line feeds, to start a new paragraph you need a blank line

Reply:
to do a single spaced line break takes a bit of effort...the line you want to keep together must have a verical bar at the beginning...if the line needs an indent (like to keep it within the note block) you indent first, then vertical bar, a space and then the text...you can see it in my changes that I will do shortly